src/validation/services/subject.js

```
@@ -0,0 +1,66 @@
+      error.status = 400;
+      throw error;
+    }
+
+    if (name.trim().length < 2 || name.trim().length > 100) {
+      const error = new Error('Name must be between 2 and 100 characters.');
+      error.status = 400;
+      throw error;
+    }
+  }
+
+  const trimmedName = filteredUpdateData.name.trim();
```

ESLint

[error] Extra semicolon.

(semi)